src/doc/unstable-book/src/language-features/intrinsics.md

```diff
@@ -53,7 +53,8 @@ Various intrinsics have native MIR operations that they correspond to. Instead o
 backends to implement both the intrinsic and the MIR operation, the `lower_intrinsics` pass
 will convert the calls to the MIR operation. Backends do not need to know about these intrinsics
 at all. These intrinsics only make sense without a body, and can be declared as a `#[rustc_intrinsic]`.
-The body is never used, as calls to the intrinsic do not exist anymore after MIR analyses.
+The body is never used as the lowering pass implements support for all backends, so we never have to
+use the fallback logic.
```

Comment:
@rust-lang/rust-analyzer I guess this PR will fail to land since RA has a copy of this file for the `hover_feature` test. How can I adjust the test to not require such a hard-to-maintain full copy?

Reply:
Is there a feature that won't really ever change maybe that we can just swap to?

Reply:
Ah well, replacing the hover test with
```rust

#[test]
fn hover_feature() {
    let (analysis, position) = fixture::position(r#"#![feature(intrinsics$0)]"#);
    analysis
        .hover(
            &HoverConfig { links_in_hover: true, ..HOVER_BASE_CONFIG },
            FileRange { file_id: position.file_id, range: TextRange::empty(position.offset) },
        )
        .unwrap()
        .unwrap();
}
```
should suffice

Reply:
As someone who has ran into tests relying on "surely this will never change" about a dozen times over the years, and each time had to do a frustrating search for what this test *actually* cared about and what was entirely incidental reliance on a feature -- I'd prefer if we didn't go down that route. Why does the test need to compare the full string for equality? There's no reason for that that I can think of.

If absolutely necessary we could probably have a dedicated feature that exists solely for the purpose of testing.

Reply:
> Ah well, replacing the hover test with

Ah, that's great, thanks a lot. :)